### PR TITLE
harfbuzz (and deps): enable for clangarm64

### PR DESCRIPTION
**Pull request by dennisameling**
As requested in https://github.com/msys2/MINGW-packages/pull/10333#issuecomment-997131846

I need `harfbuzz` for `pango`, but since it has quite a large dependency tree, I figured it'd be better to create a separate PR for this package first.

I was able to build all packages on my arm64 device, but the checks for `python-regex` fail with:

```shell
==> Starting check()...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'regex'
==> ERROR: A failure occurred in check().
    Aborting...
```

That error seems unrelated to arm64 specifically. I ran the build with `--nocheck` for now.

### CODE REVIEW

File: mingw-w64-python-pytest-runner/PKGBUILD [hunk 1]
```diff
@@ -10,14 +10,14 @@ conflicts=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
...
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
 url='https://github.com/pytest-dev/pytest-runner'
 license=('MIT')
 depends=("${MINGW_PACKAGE_PREFIX}-python"
          "${MINGW_PACKAGE_PREFIX}-python-pytest")
 makedepends=("${MINGW_PACKAGE_PREFIX}-python-setuptools"
              "${MINGW_PACKAGE_PREFIX}-python-setuptools-scm")
-#checkdepends=("${MINGW_PACKAGE_PREFIX}-python-pytest-black"
+checkdepends=("${MINGW_PACKAGE_PREFIX}-python-pytest-black")
```
Comment (dennisameling) [inline mingw-w64-python-pytest-runner/PKGBUILD:20]: This one is actually needed for the checks to pass. Have added `python-pytest-black` for arm64 as well 👍🏼

**Comment by jeremyd2019** [comment]
```
 FAILED: gir/Gio-2.0.typelib 
  "C:/msys64/clangarm64/bin/meson" "--internal" "exe" "--unpickle" "C:/_/mingw-w64-gobject-introspection/src/build-aarch64-w64-mingw32/meson-private/meson_exe_g-ir-compiler_6ec754f4debb3f248ccf54fb1fb6b9236dde118d.dat"
  while executing ['C:/_/mingw-w64-gobject-introspection/src/build-aarch64-w64-mingw32/tools/g-ir-compiler.exe', '-o', 'gir/Gio-2.0.typelib', 'gir/Gio-2.0.gir', '--includedir', 'C:/_/mingw-w64-gobject-introspection/src/build-aarch64-w64-mingw32/gir', '--includedir', 'C:/_/mingw-w64-gobject-introspection/src/gobject-introspection-1.70.0/gir']
  --- stdout ---
  
  --- stderr ---
  
  
  (g-ir-compiler.exe:9392): GLib-GIO-WARNING **: 02:11:16.828: Unexpectedly, UWP app `c5e2524a-ea46-4f67-841f-6a9465d9d515_10.0.22000.1_neutral_neutral_cw5n1h2txyewy' (AUMId `c5e2524a-ea46-4f67-841f-6a9465d9d515_cw5n1h2txyewy!App') supports 1 extensions but has no verbs
  
  
  ninja: build stopped: subcommand failed.
  ==> ERROR: A failure occurred in build().
      Aborting...
```

No idea what that is supposed to mean.

**Comment by dennisameling** [comment]
@jeremyd2019 that's weird. The problematic UWP package ID seems to belong to the "UWP File Explorer": https://www.thurrott.com/windows/windows-10/115275/windows-10-tip-unlock-uwp-file-explorer

What Windows arm64 build are you running? I'm on 22000.376.

Maybe there was some Windows Store update in progress while you attempted the build? It works correctly on my machine:

[gobject-introspection.txt](https://github.com/msys2/MINGW-packages/files/7751431/gobject-introspection.txt)
[mingw-w64-clang-aarch64-gobject-introspection-1.70.0-1-any.pkg.tar.zip](https://github.com/msys2/MINGW-packages/files/7751436/mingw-w64-clang-aarch64-gobject-introspection-1.70.0-1-any.pkg.tar.zip)

**Comment by jeremyd2019** [comment]
The runner was 22000.376, but I expect my scripts to automate setting up a runner from a fresh Windows are to blame.  I've seen that it doesn't seem to set up the "modern" apps entirely (it may be the responsibility of some background job after first logon?), and also I set the default shell to a CMD script that launches the runner, so explorer isn't actually running at the time the jobs are run.

**Comment by jeremyd2019** [comment]
It succeeds on my raspberry pi (which also isn't running explorer), so it's probably a matter of the UWP apps not being deployed into the user profile entirely.

**Comment by jeremyd2019** [comment]
Yep, need a new command added to my scripts when setting up a new Windows instance.  Unfortunately, what I found uses powershell.  That's pretty slow, especially under TCG.  I already can't really use TCG for much because it's too slow, but I sometimes test my scripts on it.

**Comment by jeremyd2019** [comment]
> Yep, need a new command added to my scripts when setting up a new Windows instance. Unfortunately, what I found uses powershell.

For the record, this was:
```xml
<CommandLine>powershell -ExecutionPolicy Bypass -Command &quot;Get-AppXPackage | Foreach {Add-AppxPackage -DisableDevelopmentMode -Register &quot;&quot;&quot;&quot;$($_.InstallLocation)\AppXManifest.xml&quot;&quot;&quot;&quot;}&quot;</CommandLine>
```

**Pull request merged**